lib/modules/manager/tekton/extract.spec.ts

```diff
@@ -9,7 +9,7 @@ describe('modules/manager/tekton/extract', () => {
         'test-file.yaml'
       );
       expect(result).toMatchSnapshot();
-      expect(result?.deps).toHaveLength(16);
+      expect(result?.deps).toHaveLength(27);
```

Comment:
Do we really need so many new fixtures? Can we remove most of the old `resource` fixtures and only leave one as they are the same as the new `params` beside the property rename?